### ISSUE TITLE
Install fails on Windows 10

### ISSUE THREAD

**Issue by Inventsable** [runegan/jsxbin#17]
Hello. I've installed this previously with no issue back when it was still using the ESTK, and recently came to revisit it but get an error when running `npm install jsxbin`:

```bash
> jsxbin@2.0.1 postinstall C:\Users\TRSch\OneDrive\Documents\Dev\NPMpackages\bombino-commands\node_modules\jsxbin
> tar -xzf esdebugger-core.tar.gz

esdebugger-core/mac/Frameworks/esdcore.framework/Resources: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Resources'
esdebugger-core/mac/Frameworks/esdcore.framework/esdcore: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\esdcore'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/Current: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\Current'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/Resources: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\Resources'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/adobe_caps: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\adobe_caps'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/Versions/Current: Can't create '\\\\?\\C:\\Users\\TRSch\\OneDrive\\Documents\\Dev\\NPMpackages\\bombino-commands\\node_modules\\jsxbin\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\Versions\\Current'
tar: Error exit delayed from previous errors.
npm WARN bombino-commands@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! jsxbin@2.0.1 postinstall: `tar -xzf esdebugger-core.tar.gz`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jsxbin@2.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\TRSch\AppData\Roaming\npm-cache\_logs\2019-11-15T01_18_00_267Z-debug.log

```

Noticeably looks like it's trying to grab or do something with a Mac file structure and isn't reading my OS prior to know I'm using Windows. Any workaround to this?

**Comment by mzuzek** [comment]
Just hit the same problem - **tar** fails npm on Win10 (both _cmd_ and _PS_):
```shell
PS D:\jsxbin.local> tar -xzf esdebugger-core.tar.gz
```
fails with:
```shell
esdebugger-core/mac/Frameworks/esdcore.framework/Resources: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Resources'
esdebugger-core/mac/Frameworks/esdcore.framework/esdcore: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\esdcore'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/Current: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\Current'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/Resources: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\Resources'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/adobe_caps: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\adobe_caps'
esdebugger-core/mac/Frameworks/esdcore.framework/Versions/A/Resources/adobe_caps.framework/Versions/Current: Can't create '\\\\?\\D:\\jsxbin.local\\esdebugger-core\\mac\\Frameworks\\esdcore.framework\\Versions\\A\\Resources\\adobe_caps.framework\\Versions\\Current'
tar.exe: Error exit delayed from previous errors.
```

One workaround would be to:
1. clone repo locally to ```jsxbin.local```
2. disable **tar** in [package.json:29](https://github.com/runegan/jsxbin/blob/18f4c76010e8c2bca9360b7b839761cd74a08d2f/package.json#L29) setting ```"postinstall": ""```
3. manually unpack ```esdebugger-core.tar.gz``` to replace it with ```esdebugger-core``` folder.
4. Install from local folder: ```npm install jsxbin.local```

**Comment by mzuzek** [comment]
#### Problem cause
`esdebugger-core.tar.gz` contains folders **named with colon** (like ```Resources:``` or ```esdcore:```, see errors in my previous comment) which are treated specially on Windows, see https://superuser.com/questions/730408/how-to-extract-tar-file-on-windows-when-the-filenames-include-a-colon.

#### Suggested solutions
* Fix the archive (remove colons from folder names) or create separate, platform specific archives.
* Fix postinstall __tar__ command to handle colons on Windows properly (rename on the fly ?).

**Comment by vj-ome** [comment]
I've  got the same problem. The tar file contains the symlinks for the mac directories, but the windows tar command does not support this symlink handling. 

Suggested solutions:
* Don't use symlinks anymore for a multi platform tar 
* or use two different files: one for windows and one for mac and just decompress the needed one.
* or just use the plain original folder
Maybe the old solution from the 2.0.0 version would be simplest one.
Maybe you should use some multi platform test environments.

**Comment by sammarks** [comment]
Another workaround is to use something like Cygwin to run the installation instead of running it in Windows by itself. That's what I was using when developing the original PR.

And yep it looks like @MikolajZuzek has the best solution. We just need to create a new archive that's windows-specific and update the `postinstall` scripts to call the proper one.

**Comment by runegan** [comment]
Fixed in #22
